copy-addresses.py

```diff
@@ -0,0 +1,42 @@
+import json
```

Comment:
Python is nice for tasks like this but it would have been nice to avoid a new language in the monorepo :)

Reply:
yeah might just close this PR, I don't think we will need this anytime soon